lib/plaid/products/processor.rb

```diff
@@ -126,6 +126,37 @@ def create(access_token, account_id)
     subproduct :processor_token, ProcessorToken
   end
 
+  # Public: Class used to call the generic ProcessorToken sub-product.
+  class ProcessorToken < BaseProduct
+    # Public: Creates a processor token from an access_token.
+    #
+    # Does a POST /processor/token/create call which can be used to
+    # generate any non-stripe processor token for a given account ID.
+    # If the processor is Stripe, the stripe endpoint will be called
+    # instead.
+    #
+    # access_token - access_token to create a public token for.
+    # account_id - ID of the account to create a processor token for.
+    # processor - name of the processor to create a token for.
+    #
+    # Returns a ProcessorTokenResponse object containing a processor token.
+    def create(access_token, account_id, processor)
+      endpoint = 'processor/token/create'
+      options = {
+        'access_token' => access_token,
+        'account_id' => account_id,
+        'processor' => processor
+      }
```

Comment:
We can make these symbol keys to clean this up and make them more consistent with the other processor endpoints (even though I'm pretty sure Faraday will accept both key types):

```suggestion
      options = {
        access_token: access_token,
        account_id: account_id,
        processor: processor
      }
```

Note also that we should change line 153 to `options.delete(:processor)` if we do this

Reply:
⚡️